README.md

```diff
@@ -62,10 +87,6 @@ Some “relative” colors use these colors for various components:
 
 Our dark mode is powered by [`prefers-color-scheme: dark`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme).
 
-## Custom domains
```

Comment:
I removed this block since it's not available anymore. Does that seem reasonable?